src/Controls/src/Core/Properties/AssemblyInfo.cs

```diff
@@ -5,7 +5,9 @@
 
 [assembly: InternalsVisibleTo("iOSUnitTests")]
 [assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Compatibility.ControlGallery")]
-[assembly: InternalsVisibleTo("Microsoft.Maui.Controls")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Compatibility")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Compatibility.Android")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Compatibility.iOS")]
```

Comment:
This is temporary because we now have core services inside Controls and Compatibility. The service tools in Core need to be validated and determined what is public and what is not.